engine/src/cmd/computer.cpp

```diff
@@ -14,6 +15,19 @@ float Computer::max_ab_speed() const
     return (!combat_mode) ? combat_mode_mult*max_combat_speed : max_combat_ab_speed;
 }
 
+Computer::RADARLIM::RADARLIM() {
+    static float default_max_range = GameConfig::GetVariable( "graphics", "hud", "radarRange", 20000 );
+    static float default_tracking_cone = GameConfig::GetVariable( "physics", "autotracking", 0.93 );     //DO NOT CHANGE see unit_customize.cpp
+
+    // Note: below is probably a stale comment
+    // DO NOT CHANGE see unit_customize.cpp
+    static float default_lock_cone = GameConfig::GetVariable( "physics", "lock_cone", 0.8 );
```

Comment:
Would prefer that we not use statics here; having class variables to hold the value would be better.